### ISSUE TITLE
[cocom] analysis fail with some repository

### ISSUE THREAD

**Issue by florentk** [chaoss/grimoirelab-elk#755]
I got this error for some on our repositories : 
```
2019-12-31 17:40:08 > 2019-12-31 16:40:05,843 - grimoire_elk.elk - ERROR - Error enriching ocean from cocom (https://github.com/kiwix/tools): 'message'
2019-12-31 17:40:08 > Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/grimoire_elk/elk.py", line 672, in enrich_backend
    enrich_count = enrich_items(ocean_backend, enrich_backend)
  File "/usr/local/lib/python3.5/dist-packages/grimoire_elk/elk.py", line 452, in enrich_items
    total = enrich_backend.enrich_items(ocean_backend)
  File "/usr/local/lib/python3.5/dist-packages/grimoire_elk/enriched/cocom.py", line 223, in enrich_items
    rich_items = self.get_rich_items(item)
  File "/usr/local/lib/python3.5/dist-packages/grimoire_elk/enriched/cocom.py", line 183, in get_rich_items
    eitem['message'] = entry['message']
2019-12-31 17:40:08 > KeyError: 'message'
```

https://github.com/kiwix/metrics/issues/16

**Comment by valeriocos** [comment]
Hi @florentk, sorry for the late reply!

If the repo is public, could you share it so I'll be able to replicate the issue?

**Comment by florentk** [comment]
@valeriocos yes you cas try with this https://github.com/kiwix/tools

**Comment by valeriocos** [comment]
The problem is due to a commit without message, see details below.
```
slimbook@slimbook-KATANA:/tmp/tools$ git show 899e6b21130d10811e0d9853194556e7cb4d247c
commit 899e6b21130d10811e0d9853194556e7cb4d247c
Author: coulonito <coulonito@users.sourceforge.net>
Date:   Thu Mar 1 09:34:44 2007 +0000

diff --git a/chrome/content/interfacewiki/default.css b/chrome/content/interfacewiki/default.css
new file mode 100644
index 0000000..525f345
--- /dev/null
+++ b/chrome/content/interfacewiki/default.css
@@ -0,0 +1,4 @@
+#wk-noresult {
+
+  color:#aaa;
+}
diff --git a/chrome/content/interfacewiki/img/logo.png b/chrome/content/interfacewiki/img/logo.png
new file mode 100644
index 0000000..840de7d
Binary files /dev/null and b/chrome/content/interfacewiki/img/logo.png differ
diff --git a/chrome/content/interfacewiki/interfacewiki.xul b/chrome/content/interfacewiki/interfacewiki.xul
index e721151..eaec8c6 100755
--- a/chrome/content/interfacewiki/interfacewiki.xul
+++ b/chrome/content/interfacewiki/interfacewiki.xul
@@ -14,7 +14,8 @@
        width   = "800"
        height  = "600"
        screenX ="10"
-       screenY ="10"
```

**Comment by jgbarah** [comment]
But this mean we need to ckeck if the commit has a message (or capture the exception) in GrimoireELK, right?

**Comment by valeriocos** [comment]
yes, I'm working on the fix. The Git enricher has something similar too: https://github.com/chaoss/grimoirelab-elk/blob/master/grimoire_elk/enriched/git.py#L343

**Comment by valeriocos** [comment]
Fixed with #763
